src/components/Users/ManageUsers.jsx

```diff
@@ -0,0 +1,165 @@
+
+// Custom Hook Imports
+import { useStatusNotification } from '../../hooks';
+// Context Imports
+import { UserListContext } from '../../contexts';
+// Component Imports
+import SolidSignup from './SolidSignup';
+import ManualWebIdForm from './ManualWebIdForm';
+import FormSection from '../Form/FormSection';
+
+/**
+ * ManageUsers Component - Component that allows users to manage other user's
+ * Pod URLs from a user's list stored on their own Pod
+ *
+ * @memberof Forms
+ * @name ManageUsers
+ */
+
+const formRowStyle = {
+  margin: '20px 0'
+};
+
+const textFieldStyle = {
+  margin: '8px'
+};
+
+const submitUser = async (userObject, session, podUrl) => {
+  let listUsers = await addUserToPod(session, userObject, podUrl);
+  listUsers = await getUserListActivity(session, listUsers);
+  return listUsers;
+};
+
+const notifyStartSubmission = (userObject, state, dispatch) => {
+  if (!userObject.username && !userObject.webId) {
+    runNotification(`Operation failed. Reason: No WebId provided`, 5, state, dispatch);
+    return;
+  }
+
+  dispatch({ type: 'SET_PROCESSING' });
+
+  runNotification(
+    `Adding user "${userObject.givenName} ${userObject.familyName}" to Solid...`,
+    5,
+    state,
+    dispatch
+  );
+};
+
+const notifyEndSubmission = (userObject, state, dispatch) => {
+  runNotification(
+    `User "${userObject.givenName} ${userObject.familyName}" added to Solid`,
+    5,
+    state,
+    dispatch
+  );
+
+  clearProcessing(dispatch);
+};
+
+const ManageUsers = () => {
+  const { session } = useSession();
+  const { state, dispatch } = useStatusNotification();
+  const [givenName, setGivenName] = useState('');
+  const [familyName, setFamilyName] = useState('');
+  const [username, setUsername] = useState('');
+  const [webId, setWebId] = useState('');
+  const { setUserList } = useContext(UserListContext);
+
+  const clearForm = () => {
+    setFamilyName('');
+    setGivenName('');
+    setUsername('');
+    setWebId('');
+  };
+
+  // Event handler for adding user from users list
+  const handleAddUser = async (event) => {
+    event.preventDefault();
+    let podUrl = (await getPodUrlAll(session.info.webId, { fetch: session.fetch }))[0];
+    podUrl = podUrl || session.info.webId.split('profile')[0];
+    const userObject = {
+      givenName,
+      familyName,
+      username,
+      webId
+    };
+
+    notifyStartSubmission(userObject, state, dispatch);
+    try {
+      const userList = await submitUser(userObject, session, podUrl);
+      setUserList(userList);
+    } finally {
+      notifyEndSubmission(userObject, state, dispatch);
+      clearForm();
+    }
+  };
+
+  return (
+    <FormSection
+      title="Add New User"
+      state={state}
+      statusType="Status"
+      defaultMessage="To be added..."
+    >
+      <form onSubmit={handleAddUser} style={formRowStyle} autoComplete="off">
+        {import.meta.env.VITE_EXPERIMENTAL ? (
```

Comment:
Following discussions in dev meetings, it seems best to rewrite the user signup form to automatically create a pod in a certain, preconfigured provider. That's what this button's intended purpose is, but it's not functioning yet. Leaving it in the codebase to make demos and discussion easier.